src/service/cid-impl.js

```diff
@@ -51,6 +51,8 @@ const CID_OPTOUT_STORAGE_KEY = 'amp-cid-optout';
 
 const CID_OPTOUT_VIEWER_MESSAGE = 'cidOptOut';
 
+const AMP_CID_BACKUP_PREFIX = 'amp-cid-backup-';
```

Comment:
I realized that all AMP stored value in origin are stored under the same localstorage key already. `amp-store:${origin}`. 
How important it is for vendors to access the storage backup value? Shall we reused the namespace?

Reply:
In the design doc, we stated that we wanted to give publishers the option to access (and potentially change) the backup value. I think reusing the name space is ok as long as it doesn't hinder this goal.

Reply:
That was my concern: reusing the namespace might hinder the goal. 
One need to access the data like this
```
ampStorage = JSON.parse(atob(localStorage.getItem('amp-store:myorigin')));
cidStorage = ampStorage['amp-cid-backup-scope']
```

Keeping all AMP stored value under the same key is neat but the tradeoff is that it may be confusing to developers who want to access it outside of AMP.

Reply:
I would prefer to keep it neat and rely on good documentation and examples to explain how to retrieve and change the values. Do you have a preference?

Reply:
Discussed offline: We agreed that from the feedback from the original issue thread, AMP is not responsible for changes made to the local storage values in the non-AMP case. If vendors/pubs wish to change the values, it will be up to them to correctly decode the values and then encode and store them back in the proper location. B/c this backup CID is stored under the same namespace as other information, faulty encoding could lead to information being dropped (i.e. stored consent info would be lost).